dradis/dradis-ce
# Datatables tagging

## Spec

Re-implement the multiple tagging behavior

### How to test

1. Open a project.
2. Add multiple issues to the project.
3. Visit the issues#index page.
4. Select some issues and confirm that the Tags dropdown appears in the table toolbar.
5. Select a tag in the dropdown and confirm that the Issues have their tag updated.


### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Thanks for contributing to Dradis!


### Copyright assignment

Collaboration is difficult with commercial closed source but we want
to keep as much of the OSS ethos as possible available to users
who want to fix it themselves.

In order to unambiguously own and sell Dradis Framework commercial
products, we must have the copyright associated with the entire
codebase. Any code you create which is merged must be owned by us.
That's not us trying to be a jerks, that's just the way it works.

Please review the [CONTRIBUTING.md](https://github.com/dradis/dradis-ce/blob/master/CONTRIBUTING.md)
file for the details.

You can delete this section, but the following sentence needs to
remain in the PR's description:

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List
